local-volume/helm/provisioner/templates/provisioner.yaml

```diff
@@ -0,0 +1,89 @@
+{{- $configMapName := .Values.configmap.configMapName }}
+{{- $provisionerNamespace := .Values.common.namespace }}
+{{- $provisionerName := .Values.daemonset.name }}
+{{- $imageFull := .Values.daemonset.image }}
+{{- $imagePullPolicy := .Values.daemonset.imagePullPolicy }}
+{{- $defaultMountDir := .Values.common.defaultMountDir }}
+{{- $serviceAccount := .Values.daemonset.serviceAccount }}
+{{- $kubeConfigEnabled := .Values.daemonset.kubeConfigEnabled }}
```

Comment:
Instead of a separate "Enabled" variable, can you just see if the values are set?